nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -58,6 +59,25 @@
 TModel = TypeVar('TModel')
 
 
+def is_half_range(overflow_fix: OverflowFix, qconfig: QuantizerConfig, is_first_layer: bool) -> bool:
+    """
+    Returns True if half_range parameter should be True to the following quantizer.
```

Comment:
The description says that this method works with the `half_range` parameter, but it's not.
Also, the current method implementation is not connected with its name and parameters.
I think it needs to be refactored.

Reply:
Remove this func